org.laptop.TurtleArtActivity.json

```diff
@@ -118,6 +70,14 @@
                 {
                     "type": "patch",
                     "path": "turtleblock-mime-icon-name.patch"
+                },
+                {
+                    "type": "patch",
+                    "path": "turtleblock-fix-setuptools-build.patch"
```

Comment:
The patch was accepted upstream, so, can this be dropped?

Reply:
Uh!  Wow!  It is merged by upstream quickly.

Reply:
Updated the commit.